<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetSKUWidget</class>
 <widget class="QDialog" name="SetSKUWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新品SKU录入</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/MainWindow/Resources/input.png"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9" stretch="8,5,0,1,8">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>隐形眼镜</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>品牌</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>型号</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>颜色</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>度数</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>SKU</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>数量</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QComboBox" name="glassbrand_comboBox">
                 <item>
                  <property name="text">
                   <string>米欧米</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>美目优瞳</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="glassMode_lineEdit">
                 <property name="placeholderText">
                  <string>示例 P04</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="glassColor_lineEdit">
                 <property name="placeholderText">
                  <string>示例 深蓝</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="glassDegree_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="glassSku_lineEdit"/>
               </item>
               <item>
                <widget class="QSpinBox" name="glassAmount_spinBox">
                 <property name="maximum">
                  <number>9999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>眼药水</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>品牌</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>型号</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>SKU</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>数量</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="eyedropBrand_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="eyedropMode_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="eyedropSku_lineEdit"/>
               </item>
               <item>
                <widget class="QSpinBox" name="eyedropAmount_spinBox">
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>护理液</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>品牌</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>型号</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>颜色</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>SKU</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>数量</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLineEdit" name="nursingBrand_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="nursingMode_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="nursingColor_lineEdit">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>示例 深蓝</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="nursingSku_lineEdit"/>
               </item>
               <item>
                <widget class="QSpinBox" name="nursingAmount_spinBox">
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>洗眼液</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>品牌</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>型号</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>颜色</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>SKU</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>数量</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLineEdit" name="washBrand_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="washMode_lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="washColor_lineEdit">
                 <property name="placeholderText">
                  <string>示例 深蓝</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="washSku_lineEdit"/>
               </item>
               <item>
                <widget class="QSpinBox" name="washAmount_spinBox">
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save_pushButton">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_pushButton">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="tableName_label">
         <property name="text">
          <string>tableName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
